flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/OperatorEventDispatcherImpl.java

```diff
@@ -82,6 +83,10 @@ public void registerEventHandler(OperatorID operator, OperatorEventHandler handl
         }
     }
 
+    Set<OperatorID> getRegisteredOperators() {
```

Comment:
Could we explicitly mark methods as `private/protected/public` for readability and consistency?

Reply:
As already discussed in the comment about `openGatewayAndUnmarkCheckpoint` above, the default package-private decorator is suitable for these methods.